hydro_deploy/core/src/hydroflow_crate/service.rs

```diff
@@ -304,22 +283,12 @@ impl Service for HydroflowCrateService {
 
         let formatted_defns = serde_json::to_string(&sink_ports).unwrap();
 
-        let stdout_receiver = self
-            .launched_binary
-            .as_mut()
-            .unwrap()
-            .write()
-            .await
-            .cli_stdout()
-            .await;
+        let stdout_receiver = self.launched_binary.as_deref_mut().unwrap().cli_stdout();
 
         self.launched_binary
-            .as_mut()
+            .as_deref_mut()
```

Comment:
This and the following can be just `as_deref`, I believe?

Reply:
Thanks, fixed!